patches/react-native+0.73.2+002+inputInlineTracking.patch

```diff
@@ -0,0 +1,712 @@
+diff --git a/node_modules/react-native/Libraries/Components/TextInput/AndroidTextInputNativeComponent.js b/node_modules/react-native/Libraries/Components/TextInput/AndroidTextInputNativeComponent.js
+index 4073836..95ba88e 100644
+--- a/node_modules/react-native/Libraries/Components/TextInput/AndroidTextInputNativeComponent.js
++++ b/node_modules/react-native/Libraries/Components/TextInput/AndroidTextInputNativeComponent.js
+@@ -416,7 +416,20 @@ export type NativeProps = $ReadOnly<{|
+   onSelectionChange?: ?DirectEventHandler<
+     $ReadOnly<{|
+       target: Int32,
+-      selection: $ReadOnly<{|start: Double, end: Double|}>,
++      selection: $ReadOnly<{|
++        start: Double,
++        end: Double,
++        cursorPosition: $ReadOnly<{|
++          start: $ReadOnly<{|
++            x: Double,
++            y: Double,
++          |}>,
++          end: $ReadOnly<{|
++            x: Double,
++            y: Double,
++          |}>,
++        |}>,
++      |}>,
+     |}>,
+   >,
+ 
+diff --git a/node_modules/react-native/Libraries/Components/TextInput/TextInput.d.ts b/node_modules/react-native/Libraries/Components/TextInput/TextInput.d.ts
+index 5cb6bf1..af1b199 100644
+--- a/node_modules/react-native/Libraries/Components/TextInput/TextInput.d.ts
++++ b/node_modules/react-native/Libraries/Components/TextInput/TextInput.d.ts
+@@ -435,6 +435,16 @@ export interface TextInputSelectionChangeEventData extends TargetedEvent {
+   selection: {
+     start: number;
+     end: number;
++    cursorPosition: {
++      start: {
++        x: number;
++        y: number;
++      };
++      end: {
++        x: number;
++        y: number;
++      };
++    };
+   };
+ }
+ 
+@@ -864,7 +874,28 @@ export interface TextInputProps
+    * The start and end of the text input's selection. Set start and end to
+    * the same value to position the cursor.
+    */
+-  selection?: {start: number; end?: number | undefined} | undefined;
++  selection?:
++  | {
++      start: number;
++      end?: number | undefined;
++      cursorPosition:
++        | {
++            start:
```

Comment:
Would it be better to add `?` here as well?

Reply:
We always have the coords of the rectangle. In cases where the end is undefined (couldn't get it), we still get the start point's coords, so there's always a number. At least, I haven't seen the opposite. But I can add optional chaining here, just in case.

Reply:
done